src/lsp/cobol_ptree/terms.ml

```diff
@@ -269,10 +269,42 @@ and class_ =
 
 
 and inline_call =                               (* in ancient terms: funident *)
-  {
-    call_fun: name with_loc;
-    call_args: effective_arg list;
-  }
+  | CallFunc of { (*name to avoid clash with call statement *)
+      call_fun: name with_loc;
+      call_args: effective_arg list;
+    }
+  | CallTrim of call_trim_args
```

Comment:
Here I strongly suggest to use inline records when the types are used only once. For the exported types, please add a relevant prefix to the labels.